abb_irb6640_support/package.xml

```diff
@@ -0,0 +1,41 @@
+<package>
+  <name>abb_irb6640_support</name>
+  <version>1.2.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 6640 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for ABB IRB 6640 manipulators. This currently includes the base model.
+    </p>
+    <p>
+    Joint limits and max joint velocities are based on the information in the
+    <a href="http://www.abbrobots.co.uk/en/3HAC028284-en.pdf">ABB IRB 6640 technical data sheet</a>
+    (Version: 3HAC 028284-001 Rev. N). All URDFs / XACROs are based on the default
+    motion and joint velocity limits, unless noted otherwise (ie: no support for
+    high speed joints, extended / limited motion ranges or other options).
+    </p>
```

Comment:
nitpicking perhaps, but this entire paragraph is indented incorrectly (needs two more spaces on each line).